tietokanta/src/main/resources/db/migration/R___Materiaalin_kaytto.sql

```diff
@@ -66,7 +66,7 @@ BEGIN
                      JOIN toteuman_reittipisteet tr ON tr.toteuma = t.id
                      JOIN LATERAL unnest(tr.reittipisteet) rp ON true
                      JOIN LATERAL unnest(rp.materiaalit) mat ON true
-               WHERE t.alkanut::date = pvm_
+               WHERE t.alkanut BETWEEN pvm_ AND (pvm_ + interval '1 day')::DATE
                  AND t.poistettu IS FALSE
             GROUP BY t.urakka, rp.talvihoitoluokka, mat.materiaalikoodi
```

Comment:
Tämä on funktio jota käytetään vain testidatan populointiin. Kuitenkin, joskus sitä voidaan ajaa käsin myös tuotantoa vasten, ja tällöin on ensiarvoisen tärkeää että t.alkanut kyselyn osa ottaa kiinni indeksiin. Jos kastaamme t.alkanut DATE:ksi, ei tartu indeksiin, siksi tämä muutos